client/src/models/ConfigModel.ts

```diff
@@ -3,13 +3,20 @@
 
 import { Theme } from '@fluentui/theme';
 
-type PostCallSurveyType = 'msforms' | 'thirdparty';
+export type PostCallSurveyType = 'msforms' | 'thirdparty';
 interface MSFormsSurveryOptions {
   surveyUrl: string;
 }
 interface ThirdPartySurveyOptions {
   surveyUrl: string;
 }
+
+export interface PostCallConfig {
+  survey?: {
+    type: PostCallSurveyType;
+    options: MSFormsSurveryOptions | ThirdPartySurveyOptions;
```

Comment:
typo, there is an extra "r" in the survey
should be MsFormsSurveyOptions

Reply:
This is done